test/trans_interop.py

```diff
@@ -39,40 +99,105 @@ def verify_transaction_amount_confirm(self, nodeTxOut, nodeTxIn, amount):
     print("### Balance of Spending Node(%s) " %debit_before)
     print("### Balance of Receiving Node(%s) " %credit_before)
     
-    
     txid = self.nodes[nodeTxOut].sendtoaddress(self.nodes[nodeTxIn].getnewaddress(), amount)
     print("=> txid :  %s " % txid)
     self.sync_all()
-    
-    #print(self.nodes[nodeTxOut].listtransactions())
-    print("### Balance (Before) : %s" % [ x.getinfo()["balance"] for x in self.nodes])  
-    assert_array_result(self.nodes[nodeTxOut].listtransactions(),
-                        {"txid":txid},
-                        {"category":"send","account":"","amount":Decimal(str(-amount)),"confirmations":0})                 
-    assert_array_result(self.nodes[nodeTxIn].listtransactions(),
-                        {"txid":txid},
-                        {"category":"receive","account":"","amount":Decimal(str(amount)),"confirmations":0})
+
+    if (nodeTxOut != nodeTxIn):
+        #print(self.nodes[nodeTxOut].listtransactions())
+        print("### Balance (Before) : %s" % [ x.getinfo()["balance"] for x in self.nodes])  
```

Comment:
let's use the logging.info("string") rather than print.  This will let us turn it off during automated testing.

Reply:
yes so much nicer with logging.info
I have replaced all print with logging.info